src/coreclr/jit/emit.cpp

```diff
@@ -4189,6 +4189,12 @@ void emitter::emitDispCommentForHandle(size_t handle, size_t cookie, GenTreeFlag
             printf("%s %s for %s", commentPrefix, flag == GTF_ICON_STATIC_HDL ? "data" : "box", fieldName);
             return;
         }
+
+        if (flag == GTF_ICON_STATIC_ADDR_PTR)
+        {
+            printf("%s static fld addr", commentPrefix);
```

Comment:
```suggestion
            printf("%s static base addr cell", commentPrefix);
```
(I missed this one.)